source/documentation/upgrade_guide/topics/Installing_Virtual_Product_Hypervisors_on_local_RHEL.adoc

```diff
@@ -0,0 +1,88 @@
+[id="Installing_virtual_product_hypervisors_on_local_RHEL_{context}"]
+= Installing {virt-product-shortname} hypervisors from a local_RHEL repository
+
+If your system uses a private {virt-product-fullname} ({virt-product-shortname}) environment, but without Red Hat Satellite, you might need to install {virt-product-shortname} hypervisors (RHV-H) from a repository hosted on a local RHEL system instead of the Red Hat hosted Content Delivery Network (CDN).
+
+.Procedure
+. On the system hosting the offline repository, create a file `/etc/yum/repos.d/rhvh-mirror.repo` similar to:
++
+[options="nowrap" subs="normal"]
+----
+[rhvh-4-for-rhel-8-x86_64-rpms]
+name = Red Hat Virtualization Host for RHEL 8 x86_64 (RPMs)
+baseurl = https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/rhvh/4/os
+enabled = 0
+gpgcheck = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslverify = 1
+sslcacert = /etc/rhsm/ca/redhat-uep.pem
+metadata_expire = 86400
+enabled_metadata = 1
+sslclientcert =
+sslclientkey =
+----
++
+You need to populate the `sslclientcert` and `sslclientkey` fields with full pathnames to the correct files containing the appropriate certificate and key. The `/etc/pki/entitlement` directory contains one or more pair of certificate and key files, but only one pair contains the RHV-H entitlement you need.
+
+. To find the certificate file:
+
+.. List all files in the `/etc/pki/entitlement` directory:
++
+[options="nowrap" subs="normal"]
+----
+[root@rhel8testvm entitlement]# cd /etc/pki/entitlement/
+[root@rhel8testvm entitlement]# ls -al
+total 836
+drwxr-xr-x.  2 root root    202 May 28 15:18 .
+drwxr-xr-x. 15 root root    208 Apr 23  2020 ..
+-rw-r--r--.  1 root root   3243 May 28 15:18 4522783034260408538-key.pem
+-rw-r--r--.  1 root root 152622 May 28 15:18 4522783034260408538.pem
+-rw-r--r--.  1 root root   3243 May 28 15:18 5659494963772844103-key.pem
+-rw-r--r--.  1 root root 343394 May 28 15:19 5659494963772844103.pem
+-rw-r--r--.  1 root root   3243 May 23 13:19 645832581386032208-key.pem
+-rw-r--r--.  1 root root 343389 May 23 13:19 645832581386032208.pem
+[root@rhel8testvm entitlement]#
+----
+
+.. Use the `rct cat-cert` command on each certificate in order to find the one that contains the RHV-H entitlement:
++
+[options="nowrap" subs="normal"]
+----
+[root@rhel8testvm entitlement]# cd /etc/pki/entitlement/
+[root@rhel8testvm entitlement]# rct cat-cert 5659494963772844103.pem | grep rhvh/4/ | grep URL
+        URL: /content/beta/rhel/server/7/$basearch/rhvh/4/os
+        URL: /content/dist/rhel/server/7/7Server/$basearch/rhvh/4/os
+        URL: /content/beta/layered/rhel8/x86_64/rhvh/4/os
+        URL: /content/dist/layered/rhel8/x86_64/rhvh/4/os
+----
+
+. Identify the correct certificate and fill in the `sslclientcert` and `sslclientkey` values in the `.repo` file above:
++
```

Comment:
IBM Style says: Do not use above to indicate a relative location
in a document, as in "the above restrictions" or "the steps above". Use "previous" or "preceding".